chrome/chrome.d.ts

```diff
@@ -2397,12 +2347,7 @@ declare module chrome.webRequest {
         formData?: Object;
     }
 
-    interface OnBeforeRequestDetails {
-        tabId: number;
-        parentFrameId: number;
-        url: string;
-        timeStamp: number;
-        frameId: number;
+    interface OnBeforeRequestDetails extends CallbackDetails {        
         requestBody: RequestBody;
```

Comment:
[want help]

(However it will never be used as params but only result passed in callback,) 
`requestBody` of `OnBeforeRequestDetail` is optional field.

https://developer.chrome.com/extensions/webRequest#event-onBeforeRequest

I think we should add `?` suffix.

@vvakame 
What do you think about 

**"Adding `?` suffix to interface which is used only as callback result"**